data/transactions/logic/opcodes.go

```diff
@@ -572,6 +607,24 @@ var OpSpecs = []OpSpec{
 	{0xad, "b^", opBytesBitXor, proto("bb:b"), 4, costly(6)},
 	{0xae, "b~", opBytesBitNot, proto("b:b"), 4, costly(4)},
 	{0xaf, "bzero", opBytesZero, proto("i:b"), 4, opDefault()},
+	multiOp(0xa0, "bmath", multiVersion, []OpSpec{
+		{0xa0, "b+", opBytesPlus, proto("bb:b"), multiVersion, costly(10)},
+		{0xa1, "b-", opBytesMinus, proto("bb:b"), multiVersion, costly(10)},
+		{0xa2, "b/", opBytesDiv, proto("bb:b"), multiVersion, costly(20)},
+		{0xa3, "b*", opBytesMul, proto("bb:b"), multiVersion, costly(20)},
+		{0xa4, "b<", opBytesLt, proto("bb:i"), multiVersion, opDefault()},
+		{0xa5, "b>", opBytesGt, proto("bb:i"), multiVersion, opDefault()},
+		{0xa6, "b<=", opBytesLe, proto("bb:i"), multiVersion, opDefault()},
+		{0xa7, "b>=", opBytesGe, proto("bb:i"), multiVersion, opDefault()},
+		{0xa8, "b==", opBytesEq, proto("bb:i"), multiVersion, opDefault()},
+		{0xa9, "b!=", opBytesNeq, proto("bb:i"), multiVersion, opDefault()},
+		{0xaa, "b%", opBytesModulo, proto("bb:b"), multiVersion, costly(20)},
+		{0xab, "b|", opBytesBitOr, proto("bb:b"), multiVersion, costly(6)},
+		{0xac, "b&", opBytesBitAnd, proto("bb:b"), multiVersion, costly(6)},
+		{0xad, "b^", opBytesBitXor, proto("bb:b"), multiVersion, costly(6)},
+		{0xae, "b~", opBytesBitNot, proto("b:b"), multiVersion, costly(4)},
+		{0xaf, "bzero", opBytesZero, proto("i:b"), multiVersion, opDefault()}}...,
```

Comment:
Interesting - are you using `0xa*` in the children to intentional make all the bytemath opcodes use `0xa` a lot?